src/from_parentdir.py

```diff
@@ -19,8 +19,8 @@ def versions_from_parentdir(parentdir_prefix, root, verbose):
         root = os.path.dirname(root)  # up a level
 
     if verbose:
-        print("Tried directories %s but none started with prefix %s" %
-              (str(rootdirs), parentdir_prefix))
+        print(f"Tried directories {str(rootdirs)} "
+              f"but none started with prefix {parentdir_prefix}")
```

Comment:
`str()` is called implicitly.

```suggestion
        print(f"Tried directories {rootdirs} "
              f"but none started with prefix {parentdir_prefix}")
```